Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.Model.sb.in

```diff
@@ -25,3 +25,27 @@
 
 (allow mach-lookup
     (global-name "com.apple.CARenderServer"))
+
+(allow iokit-open-user-client
+    (iokit-user-client-class "AppleParavirtDeviceUserClient"))
+(allow iokit-open-user-client
+    (iokit-user-client-class "AGXDeviceUserClient"))
+
+(allow mach-lookup
+    (global-name "com.apple.audio.AudioSession"))
+(allow mach-lookup
+    (global-name "com.apple.surfboard.applicationservice"))
+(allow mach-lookup
+    (xpc-service-name "com.apple.ImageIOXPCService"))
```

Comment:
More likely we should disable OOP ImageIO in this process like we do in the WP!

Reply:
This line will be removed once the ImageIO work has been done.